aws/table_aws_tagging_resource.go

```diff
@@ -27,6 +30,14 @@ func tableAwsTaggingResource(_ context.Context) *plugin.Table {
 		List: &plugin.ListConfig{
 			Hydrate: listTaggingResources,
 			Tags:    map[string]string{"service": "tag", "action": "GetResources"},
+			KeyColumns: []*plugin.KeyColumn{
+				{
+					Name:       "resource_types",
+					Require:    plugin.Optional,
+					Operators:  []string{"="},
+					CacheMatch: query_cache.CacheMatchExact,
```

Comment:
@ParthaI As far as I can tell there's no caching being applied for this query, the query times are the almost the same when you rerun a query, so it behaves very differently then when another table returns cached data. Any idea how to debug this further? Or should I just remove this line then?